client/src/featureform/dashboard_metadata.py

```diff
@@ -68,6 +71,43 @@ def version():
     return {"version": get_package_version()}
 
 
+@dashboard_app.route("/data/sourcedata", methods=["GET"])
+@cross_origin(allow_headers=["Content-Type"])
+def sourcedata():
+    limit = 150
+    n = 0
+    name = request.args["name"]
+    variant = request.args["variant"]
+    if name == "" or variant == "":
+        error = f"Error 400: GetSourceData - Could not find the name({name}) or variant({variant}) query parameters"
```

Comment:
do we need `Error 400` here since we are returning the error code?

Reply:
we don't, but I assume the user would like to see the error code without opening the console. thoughts?

Reply:
What is the norm? I would refer to your expert opinion on it.

Reply:
so the norm is not show the error code (for most web apps) but since this dashboard is a power-user tool it's more convenient for them. let's keep it for now.